landscape/explanation/glossary/ssl-cert.md

```diff
@@ -0,0 +1,7 @@
+# SSL Certificates
+
+**Secure Sockets Layer (SSL) certificates** are part of creating secure communication over the internet. They provide a process for encrypting and authenticating data sent between a client and a server. In the context of Landscape, being a server management tool, it uses SSL certificates to communicate securely with its servers, reducing the chance of unauthorized access and data breaches.
```

Comment:
```suggestion
**Secure Sockets Layer (SSL) certificates** are part of creating secure communication over the internet. They provide a process for encrypting and authenticating data sent between a client and a server. In the context of Landscape, being a systems management tool, it uses SSL certificates to communicate securely with its servers, reducing the chance of unauthorized access and data breaches.
```

Reply:
"Server" is really _wrong_, but "systems management" just feels a bit more precise and it is usually how I refer to it in the documentation

Reply:
Thank you for the comment.  Can you walk me through this part of the review process? Should I go back and edit the original file on the pull request, or use the “Commit suggestion” option below your suggested changes?

Reply:
@Owslla Just answered this in a comment below :smile: